verify/verify-flags/known-flags.txt

```diff
@@ -1,4 +1,5 @@
 apiserver-address
+access-token-dirs
```

Comment:
It's a tragedy that we need to update it every time we add a flag...
I opened https://github.com/kubernetes/perf-tests/issues/997 to address it

Reply:
Great, thanks!